docs/contributing.md

```diff
@@ -10,37 +10,24 @@ All our code is in one repository (monorepo). We currently use the pull request
     - `git checkout develop`
     - `git pull`
     - `git checkout -b branch-name`
-2. Branches ought to be named with the `{component}-{type}-{slug}` format, where component is either `frontend` or `backend`, the type is `feature`, `bugfix`, etc; and slug is a brief name for the branch, for example:
-    - `frontend-feature-avatar-component`
-    - `backend-bugfix-email-html-escaping`
-3. Work on the new branch, feel free to commit regularly. Ideally a commit should make one change to the code but the code should compile and run both before and after the change (though this is not always possible).
-4. Push the new branch to GitHub, and open a Pull Request (PR). If your branch is ready to be merged, pending review, prepend `[MRG]` to the pull request title. If it’s still work in progress, prepend `[WIP]`.
-5. When you are ready for a review, select a reviewer and ask them to review the code. Feel free to choose someone you know can review it, or if you don’t know who else, select the maintainer, or ping the appropriate channel on Slack.
+2. Branches ought to be named with the `{component}/{type}/{slug}` format, where component is either `frontend` or `backend`, the type is `feature`, `bugfix`, `refactor`, etc; and slug is a brief name for the branch, for example:
+    - `frontend/feature/avatar-component`
+    - `backend/bugfix/email-html-escaping`
```

Comment:
This contradicts "Note on slugs" in cicd.md which include frontend-feature in the slug as well.

Reply:
The slugs are generated by the CI system from the branch names, which replaces slashes with dashes. Maybe worth clarifying.